src/styles/themes/input.ts

```diff
@@ -0,0 +1,195 @@
+import { fonts } from './fonts'
+
+export type Input = typeof input
+
+export const input = {
+  chat: {
+    large: `
+        &>input: {
+            height: 48px;
+            width: 639px;
+            padding: 12px 60px 12px 20px;
+
+            &::placeholder{
+               ${fonts.body01R} color: #929399;
+            }
+            $:hover{
+                ${fonts.body01R} color: #2F2E36;
+            }
+            &:focus{
+                ${fonts.body01R} color: #2F2E36;'
+            }
+        }
+    }`,
+
+    small: `
+        &>input: {
+            height: 40px;
+            width: 328px;
+            padding: 10px 52px 10px 12px;
+
+            &::placeholder{
+                ${fonts.body02R} color: #929399;
+            }
+            $:hover{
+                ${fonts.body02R} color: #2F2E36;
+            }
+            $:focus{
+                ${fonts.body02R} color: #2F2E36;
+            }
+        },
+    `
+  },
+  default: {
+    large: {
+      default: `
+        &>input: {
+            margin: 8px 0;
+            width: 328px;
+            height: 54px;
+            border: none;
+            padding: 16px 12px 16px 12px;
+            &::placeholder{
+                ${fonts.subtitle01M} color: #929399;
```

Comment:
```suggestion
                ${fonts.subtitle01M};
                color: #929399;
```
; 빠지면 color가 반영 안될 수도 있어서 붙이는 게 좋을 것 같아요!
color도 color 객체로 부탁드림니둥..!

Reply:
오 저 fonts 객체안에 ; 가 있는데 쓰면 안되지않을까해서 빼놨는데 써야하나요...!?

```
export const fonts = {
  body01B:
    'font-size: 16px; font-weight: bold; line-height: 24px; letter-spacing: -0.1px;',
  body01M:
    'font-size: 16px; font-weight: 500; line-height: 24px; letter-spacing: -0.1px;',
  body01R:
    'font-size: 16px; font-weight: normal; line-height: 24px; letter-spacing: -0.1px;',
  body02B:
    'font-size: 14px; font-weight: bold; line-height: 24px; letter-spacing: -0.1px;',
  body02L:
    'font-size: 14px; font-weight: normal; line-height: 26px; letter-spacing: -0.1px;',
  body02M: 'font-size: 14px; font-weight: 500; line-height: 20px;',
  body02R:
    'font-size: 14px; font-weight: normal; line-height: 20px; letter-spacing: -0.1px;',
  caption01B:
    'font-size: 12px; font-weight: bold; line-height: 16px; letter-spacing: -0.05px;',
  caption01M:
    'font-size: 12px; font-weight: 500; line-height: 16px; letter-spacing: -0.05px;',
  display01B:
    'font-size: 36px; font-weight: bold; line-height: 40px; letter-spacing: -0.1px;',
  display02B: 'font-size: 28px; font-weight: bold; line-height: 28px;',
  display02M: 'font-size: 28px; font-weight: 500; line-height: 28px;',
  headline01B: 'font-size: 24px; font-weight: bold; line-height: 32px;',
  headline02B:
    'font-size: 20px; font-weight: bold; line-height: 28px; letter-spacing: -0.1px;',
  subtitle01B:
    'font-size: 18px; font-weight: bold; line-height: 22px; letter-spacing: -0.1px;',
  subtitle01M:
    'font-size: 18px; font-weight: 500; line-height: 22px; letter-spacing: -0.05px;'
} as const

```